deb-get

```diff
@@ -471,7 +471,8 @@ function validate_deb() {
 
         if { { { [ "${METHOD}" == github ] || [ "${METHOD}" == website ]; } && [ -s "${CACHE_FILE}" ]; } || [ "${METHOD}" == direct ]; } &&
            { [ "${ACTION}" != "prettylist" ] && [ "${ACTION}" != "remove" ] && [ "${ACTION}" != "purge" ]; } &&
-           { [ -z "${URL}" ] || [ -z "${VERSION_PUBLISHED}" ]; }; then
+           { [ -z "${URL}" ] || [ -z "${VERSION_PUBLISHED}" ]; } &&
+           { [[ " ${ARCHS_SUPPORTED} " =~ " ${HOST_ARCH} " ]]; }; then
```

Comment:
I think it would be necessary/desirable to add codename support check here too: 
```
 { [[ " ${ARCHS_SUPPORTED} " =~ " ${HOST_ARCH} " ]]; }  && { [ -z "${CODENAMES_SUPPORTED}" ] || [[ " ${CODENAMES_SUPPORTED} " =~ " ${UPSTREAM_CODENAME} " ]]; }; then
```